pkg/baseline/packageIndex.json

```diff
@@ -9,8 +9,9 @@
       "BaselineVersion": "4.4.0",
```

Comment:
It doesn't help explain but I just updated this locally and it worked, as in no validation errors.